generators/server/templates/dotnetcore/src/Project.Infrastructure/Data/ApplicationDatabaseContext.cs.ejs

```diff
@@ -61,5 +80,38 @@ namespace <%= namespace %>.Infrastructure.Data {
                 .OnDelete(DeleteBehavior.Cascade);
             <%_ } _%>
         }
+
+        /// <summary>
+        /// SaveChangesAsync with entities audit
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public override async Task<int> SaveChangesAsync(CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var entries = ChangeTracker
+              .Entries()
+              .Where(e => e.Entity is IAuditedEntityBase && (
```

Comment:
Here AuditedEntityBase class can be used instead of checking for the interface

Reply:
Interface is used also for User entity as we can't use multiple extends class for User entity

Reply:
> Interface is used also for User entity as we can't use multiple extends class for User entity

O